src/app/page.tsx

```diff
@@ -12,9 +12,16 @@ export default function RootLayout() {
     router.push(`/result?score=${battlePower}`);
   };
   return (
-    <div className="flex flex-col items-center justify-center min-h-screen p-4">
-      <h1 className="text-3xl font-bold mb-6 text-blue-500">SNSスカウター</h1>
-      <Form onSubmit={handleSubmit} />
+    <div className="min-h-screen flex flex-col items-center justify-center p-4 bg-gradient-to-b from-gray-900 to-black relative overflow-hidden">
+      {/*背景エフェクト */}
+      <div className="absolute inset-0 z-0">
+        <div className="grid-lines"></div>
+        <div className="scanline"></div>
+        <div className="particles"></div>
+      </div>
+      <div className="flex flex-col items-center justify-center min-h-screen p-4">
+        <Form onSubmit={handleSubmit} />
+      </div>
     </div>
```

Comment:
ここら辺、背景を持つ外側の div と、フォームをラップする内側の div で、中央揃えのスタイル (flex, items-center, justify-center, min-h-screen, p-4) が重複しているように見えるので、こんな感じにするとシンプル & ついでにフォームが後ろにいく問題が直ると思います！
```suggestion
    <div className="relative min-h-screen flex flex-col items-center justify-center p-4 bg-gradient-to-b from-gray-900 to-black overflow-hidden">
      {/* 背景エフェクト */}
      <div className="absolute inset-0 z-0">
        <div className="grid-lines"></div>
        <div className="scanline"></div>
        <div className="particles"></div>
      </div>

      <div className="relative z-10">
        <Form onSubmit={handleSubmit} />
      </div>
    </div>
```